src/messenger-shortcuts.js

```diff
@@ -112,7 +139,8 @@ document.body.onkeydown = function(event) {
     case getCode(KEYS.SEARCH_IN_CONVO):
       searchInConversation();
       break;
-    case getCode(KEYS.HELP):
+    case 111: // divide (on num keyboard)
+    case 191: // forward slash (on std. eng keyboard)
```

Comment:
Is this equivalent to `getCode(KEYS.HELP)`? If so, can you just use `getCode(KEYS.HELP)` here for consistency?

Reply:
Never mind, this makes sense now.